tests/test_addons.py

```diff
@@ -128,53 +126,45 @@ def configure_addon(self):
         self.node_addon.save()
 
     def build_url(self, **kwargs):
-        options = dict(
+        options = {'payload': jwt.encode(dict(dict(
```

Comment:
For added security set expiration to 30s (https://github.com/jpadilla/pyjwt#expiration-time-claim)